### ISSUE TITLE
Any news about the new pcb board ?

### ISSUE THREAD

**Issue by bobaxxx** [DeskPi-Team/deskpi#38]
Hi,

We dont have any news about new pcb board. Deskpi team doesnt answer to mail too.

i need also another USB3 rear adaptater

Can you make an answer for that

thx

**Comment by nitz** [comment]
I'm wondering the same. I came here to ask this yesterday, but never actually created the issue, so I'm glad to +1!

**Comment by tedder** [comment]
on 3/17 they replied to an email from me and said:
> There are some delay.
>
> The new PCB Board is in mass production.
>
> We will arrange shipments from the end of March according to the order date.

**Comment by bobaxxx** [comment]
You have pretty more Luck than me because i sent 3 mails Without a single answer ...

**Comment by jojobrogess** [comment]
Yeah the website was silently updated from "middle of March" to "after the end of March" without any notification or courtesy email's. IMO, that's extremely upsetting business practices. And the sparse, broken, or non-existent communication for months and then the shutdown for Chinese New Years or whatever Holliday it is, is extremely pathetic. 3 months with a handful of comments and an email or two to fix a pcb board. For an issue that should have been caught PRIOR to development, because that chip had WELL-KNOWN problems for SBC's. 
I get it though, they wanted it out before the m.2 argon or just faster in general, so things can happen. I get it. 
But STILL ZERO communication on what's going on, any type of eta, or even a "hey! We are still here! Working hard." That's what gets me the most. It would take less than 5 minutes to (weekly?hell monthly) post an update for better customer relations. But no? Do we not deserve it? Or is it just too much to ask for? Are you working _that_ hard that you can't give us an update? 
So yeah, I'm like a week or two away from requesting a refund because I was promised a fixed PCB board by mid march, and if that won't work then a fraudulent charge will be enacted because I purchased a product/service and the agreed upon product was not fully provided. 
It might seem like I'm salty, but it literally takes less than 5 minutes to post an update. We all have cell phones and internet connectivity. So that shows me how little they care

**Comment by nitz** [comment]
I may be wrong, but I doubt it's that they don't care. I agree an update seems like a minimal amount of effort, but having been on small-team projects where people are waiting to hear status updates, the longer it goes without one the tougher it gets to tell folx it's delayed (or worse, that you don't know how long it will take.) I imagine cultural differences might make a difference too.

I'm sure the CNY exacerbated the issue, but the silicon supply chain problems right now are ridiculously painful. At work, we have some chips on order that we have used for years that have a lead time of 42+ weeks now. They usually are an easy overnight from Digi-Key, but _everything_ is backed up.

Checking out [the website](https://deskpi.com/products/deskpi-pro-for-raspberry-pi-4) though, It looks like they posted an update on March 26. I've duplicated it below

---

**Updated News on 26th March, 2021:**
We will start to re-sent of the updated replacement PCB board according to the order date from this week.

**Resent List:**
1. USB adapter
2. Daughter Board
3. T-shaped isolation foam
4. U-shaped support frame

Please kindly keep patience, you will receive your new PCB Boards soon.

**For new orders:**

We will start shipping in mid-April.

Thanks a lot for your all support and understanding!

**We will also send all of you a DeskPi bracket for free!**

![](https://cdn.shopifycdn.net/s/files/1/0460/0325/7500/files/QQ_20210326173643_480x480.jpg?v=1617007823)
![](https://cdn.shopifycdn.net/s/files/1/0460/0325/7500/files/QQ_20210326173650_480x480.jpg?v=1617007843)

**Comment by Mister-42** [comment]
I also interested in what is NOT there.

No front IR receiver???

**Comment by Pi-Mania** [comment]
Received tracking and can confirm my replacements are on the way. (AUS Reseller) 👍

**Comment by M-H-010** [comment]
@Pi-Mania, you are lucky, Sales at Deskpi stopped responding to me as they clearly did not met the mid or end March deadline.
And as jojobrogess mentioned, an update on their website would take so little time and make a more mature impression.
I just hope the new PCB's do really solve all the issues....
Mine just develloped a new one:, undervoltage warnings and unexpexted reboots. :-(

**Comment by bpicon** [comment]
New PCB received and installed. The bridge is identified as 174c: 1153, ASMedia Technology Inc. ASM1153 SATA 3Gb / s bridge (not 1153E SATA 6Gb / s bridge)
At least in my case (SATA XRayDisk SSD 240 GB connected, twister os 2.0.1, kernel 5.10.17-v7l +) using usb-storage driver by default, not UAS driver.
I don't know what firmware the bridge uses.

pi@raspberrypi:~ $ lsusb
Bus 003 Device 003: ID 1a86:7523 QinHeng Electronics HL-340 USB-Serial adapter
Bus 003 Device 002: ID 05e3:0610 Genesys Logic, Inc. 4-port hub
Bus 003 Device 001: ID 1d6b:0002 Linux Foundation 2.0 root hub
Bus 002 Device 002: ID 174c:1153 ASMedia Technology Inc. ASM1153 SATA 3Gb/s bridge
Bus 002 Device 001: ID 1d6b:0003 Linux Foundation 3.0 root hub
Bus 001 Device 002: ID 2109:3431 VIA Labs, Inc. Hub
Bus 001 Device 001: ID 1d6b:0002 Linux Foundation 2.0 root hub
pi@raspberrypi:~ $ lsusb -t
/:  Bus 03.Port 1: Dev 1, Class=root_hub, Driver=dwc2/1p, 480M
    |__ Port 1: Dev 2, If 0, Class=Hub, Driver=hub/4p, 480M
        |__ Port 1: Dev 3, If 0, Class=Vendor Specific Class, Driver=ch341, 12M
/:  Bus 02.Port 1: Dev 1, Class=root_hub, Driver=xhci_hcd/4p, 5000M
    |__ Port 1: Dev 2, If 0, Class=Mass Storage, Driver=usb-storage, 5000M
/:  Bus 01.Port 1: Dev 1, Class=root_hub, Driver=xhci_hcd/1p, 480M
    |__ Port 1: Dev 2, If 0, Class=Hub, Driver=hub/4p, 480M

**Comment by Pi-Mania** [comment]
Here's what i know:

1. The old version of the DeskPi Pro disk chip DID support UASP, but it does not support USB booting well. (low/poor  compatibility)  The new version of the disk chip does NOT support UASP and TRIM. (DeskPi Pro V2) but has HIGH compatibility.
2. asm1153 does not support 6Gbps,only supports 3Gbps. Since the 6GB chip is currently unavailable it had to be replaced with a 3GB chip. 

maybe not what some of you wanted to hear but, i still like this DeskPi case a lot. If the 6gb chip becomes available, perhaps they will switch to that? With new V2 boards i can now boot without quirks and i get slight speed improvements. I can also boot & read from sata/ssd drive attached to rear external USB 3 port. I could not personally do this with old boards. 

Regards

**Comment by bpicon** [comment]
I don't know if I'm wrong, but according to the manufacturer's specifications (AsMedia) there is UASP support for this chip. Perhaps a firmware update can activate this support:

https://www.asmedia.com.tw/product/995Yq54sX2YI1th2/d1Eyq85QN8GhBwRC

- Compliant with USB3.0 Specification Revision 1.0
- Compliant with USB Specification Revision 2.0
- Support USB Super-Speed, High-Speed and Full-Speed Operation
- Support Mass Storage Class, Bulk-Only Transport Specification Revision 1.3
- **Support USB Attached SCSI Protocol Specification Revision 1.0**
- Compliant with Serial ATA Specification Revision 2.6Serial ATA bus up to 3Gbps Signal bandwidth
  ...
--

**Comment by Station010** [comment]
Thanks to @Pi-Mania and @bpicon for the updated information.
A good spot on the asmedia page re USAP support but I wonder why they would go to all the effort of re-creatng the board and not test the frmware is up to date and gives the support for USAP and TRIM. Cheap USB adapter cases with ASMEDIA asm1153E chips come with the support ready to use out of the box.
So unless a firmware update and instructions are suppplied with the new board in the near furture I guess its a no sale for me,
what a disappointment from a normally very reliable company.

**Comment by csmale** [comment]
This is now on the News page of the website:

> The Old Version DeskPi supports UASP. 
> The New Version DeskPi Pro on sale does not support UASP or TRIM.

Looks like a backward step if they are admitting that.

**Comment by Mister-42** [comment]
That's the final straw. I have requested a refund.

**Comment by mjeshurun** [comment]
Hello everyone, 
Sorry if my question is Noobiesh, but I'm currently running my Raspberry Pi 4 using a [Sandisk Extreme Portable 500GB SSD](https://shop.westerndigital.com/products/portable-drives/sandisk-extreme-usb-3-1-ssd#SDSSDE60-250G-G25) 
which appers to support UAS:
```
pi@raspberrypi:~ $ lsusb
Bus 002 Device 003: ID 8564:4100 Transcend Information, Inc. 
Bus 002 Device 002: ID 0781:558c SanDisk Corp. 
Bus 002 Device 001: ID 1d6b:0003 Linux Foundation 3.0 root hub
Bus 001 Device 005: ID 1b1c:1b6d Corsair 
Bus 001 Device 004: ID 0658:0200 Sigma Designs, Inc. Aeotec Z-Stick Gen5 (ZW090) - UZB
Bus 001 Device 003: ID 8564:4100 Transcend Information, Inc. 
Bus 001 Device 002: ID 2109:3431 VIA Labs, Inc. Hub
Bus 001 Device 001: ID 1d6b:0002 Linux Foundation 2.0 root hub
pi@raspberrypi:~ $ lsusb -t
/:  Bus 02.Port 1: Dev 1, Class=root_hub, Driver=xhci_hcd/4p, 5000M
    |__ Port 1: Dev 2, If 0, Class=Mass Storage, Driver=uas, 5000M
    |__ Port 2: Dev 3, If 0, Class=Hub, Driver=hub/4p, 5000M
/:  Bus 01.Port 1: Dev 1, Class=root_hub, Driver=xhci_hcd/1p, 480M
    |__ Port 1: Dev 2, If 0, Class=Hub, Driver=hub/4p, 480M
        |__ Port 2: Dev 3, If 0, Class=Hub, Driver=hub/4p, 480M
            |__ Port 1: Dev 4, If 1, Class=CDC Data, Driver=cdc_acm, 12M
            |__ Port 1: Dev 4, If 0, Class=Communications, Driver=cdc_acm, 12M
            |__ Port 3: Dev 5, If 0, Class=Human Interface Device, Driver=usbhid, 12M
            |__ Port 3: Dev 5, If 1, Class=Human Interface Device, Driver=usbhid, 12M
            |__ Port 3: Dev 5, If 2, Class=Human Interface Device, Driver=usbhid, 12M
```
and here are the hdparm test results for the Sandisk Extreme SSD:
```
pi@raspberrypi:~ $ sudo hdparm -Tt /dev/sda

/dev/sda:
 Timing cached reads:   1512 MB in  2.00 seconds = 756.64 MB/sec
 Timing buffered disk reads: 922 MB in  3.00 seconds = 307.03 MB/sec
```

My initial plan was to use the DeskPi Pro with a [Transcend 830S M.2 SSD SATA III 2TB](https://www.transcend-info.com/Products/No-982).
Based on your experience, will the Transcend 830S run in the same/slower/faster speeds as the Sandisk Extreme SSD? 
What else should I consider now that we know that the new Deskpi Pro version will lack UASP/TRIM support? 
I'd like to know if lack of UASP/TRIM are really "deal breakers"?
I guess, if my system will run at the same speeds as it runs today with the Sandisk Extreme SSD, I might still consider purchasing the Deskpi, but if the Deskpi ends up running slower, then there is no point of changing my current setup.

**Comment by Pi-Mania** [comment]
I am on V2 hardware on a Samsung 750EVO 128gb SSD.  Timing cached reads:   1620 MB in  2.00 seconds = 809.81 MB/sec
 Timing buffered disk reads: 676 MB in  3.00 seconds = 225.25 MB/sec. My TechData m.2 is pretty much same result.

**Comment by mjeshurun** [comment]
> I am on V2 hardware on a Samsung 750EVO 128gb SSD. Timing cached reads: 1620 MB in 2.00 seconds = 809.81 MB/sec
> Timing buffered disk reads: 676 MB in 3.00 seconds = 225.25 MB/sec. My TechData m.2 is pretty much same result.

Thank you @Pi-Mania. 
From your speed test, am I correct to assume I should expect similar speeds to those I have today with the Sandisk Extreme SSD?

**Comment by Aasikki** [comment]
I'm planning to use this case for my home assistant server. What's  your opinion should I cancel my order because of this or will it be a big deal for that use?

> 
> 
> This is now on the News page of the website:
> 
> > The Old Version DeskPi supports UASP.
> > The New Version DeskPi Pro on sale does not support UASP or TRIM.
> 
> Looks like a backward step if they are admitting that.

**Comment by Mister-42** [comment]
The lackof TRIM support will slow down SSDs and degrade their lifespan faster.

**Comment by Aasikki** [comment]
Not sure if I want to keep waiting for my deskpi or should I just keep my current mess with external ssd enclosure and 3d printed case with cables coming from two different sides.Sigh... I wish there were other cases like this with proper ssd support.

**Comment by risa2000** [comment]
@Aasikki FWIW I was planning to use desk-pi case for HA too, and since it did not work, I went ahead with Argon M.2 case. The major bummer was finding out that my Zigbee, Z-Wave (USB sticks) and (integrated) BT basically stopped working, because they were totally scrambled by EMI from USB 3.0 SSD traffic. So I am still running my HA from RPi 3B (no plus) with old USB 2.0. Depending on your needs and hardware you may actually have already the best setup ;).

**Comment by Aasikki** [comment]
Haha that's pretty much the exact setup I have currently. Rpi b3+ and usb 2.0 ssd. I managed to get rpi 4 for cheap though and really want to use it for home assistant and throw the old one in my 3d printer for octoprint.

**Comment by mjeshurun** [comment]
@Pi-Mania can you please comment what type of M.2 socket connector is available on the Deskpi Pro?
Is it B key, M key or B&M? 
I would like to make sure the SSD I order will fit properly.

**Comment by Aasikki** [comment]
@risa2000 Do you mean that home assistant didn't work at all with the deskpi or what went wrong? If that is the case I'm definitely going to have to cancel my order.

**Comment by risa2000** [comment]
@Aasikki No, I did not bother with desk-pi considering it did not work with USB SATA disks. My observations are based on the similar setup run on RPi 4 in Argon M.2 case, which also sports USB 3.0 to SATA interface (only to M.2 slot though).

**Comment by Aasikki** [comment]
@risa2000 ok in that case I think I'll give deskipi a go then.

**Comment by Pi-Mania** [comment]
> @Pi-Mania can you please comment what type of M.2 socket connector is available on the Deskpi Pro?
> Is it B key, M key or B&M?
> I would like to make sure the SSD I order will fit properly.

The DeskPi Case can accept either M or B+M SSD's Via a supplied daughterboard that slots into SATA port. (But NOT NVMe drives)

**Comment by mjeshurun** [comment]
Thanks @Pi-Mania 🙏
Could we potentially change the 2.5inch SSD expansion board and the M.2 connector that comes with the Deskpi Pro with a different M.2 connector board that supports SATA 6Gb, UASP and TRIM?

**Comment by Marach79** [comment]
I requested a refund and they ignored that in February. The case was useless for me and I had thrown away. Three days ago I received the replacement board. Ironic 🤣 My request to buy the missing parts.... They want to answer ASAP.... I'm still waiting. Very bad support in my opinion.

**Comment by Mister-42** [comment]
Hello @yoyojacky @deskpi 

Saying >refunded< on the order page does not make the money magically appear back on my PayPal account. Ignore emails also does not fix this. PROCEED WITH THE REFUND AT ONCE!

**Comment by Marach79** [comment]
After opening an PayPal case they want from me to send back the case to them. Sure.... paying for a not functional case and the delivery twice. Definitely no! And a support that's not able to answer ASAP.... I'm very angry about the not working communication.

**Comment by leicray** [comment]
I bought one of the new spec cases about ten days ago (from a UK supplier) and find the my Pi now freezes once or twice a day with the white disk-activity light flashing furiously. Previously, it was absolutely stable when installed in a simple case with a fan and with the SSD drive connected via a StarTech USB32SSAT3CB (Chipset: ASM1153E) USB3 to SATA adapter. That chipset supports UASP and TRIM.

Removing the drive (Kingston 120 GB SA400S37/120G) from the DeskPi Pro case and reconnecting it externally with the adapter cable to a USB3 port has restored the stability that I enjoyed previously. Boot time is subjectively faster too.

There is clearly still something wrong with the revised board. I will be requesting a refund.

**Comment by Mister-42** [comment]
> Hello @yoyojacky @deskpi
> 
> Saying >refunded< on the order page does not make the money magically appear back on my PayPal account. Ignore emails also does not fix this. PROCEED WITH THE REFUND AT ONCE!

Reminder for @yoyojacky @deskpi. I'm gonna repeat this over and over until you have done as promised.

**Comment by babyheysus** [comment]
Got my replacement board and it doesn't see my SSD at all. The pi sees the ASMedia board though. Was there anything you did differently @Pi-Mania? Outside the deskpi instructions. Everything is being pumped by a OnePlus charger so power should be good, SSD is a 120GB SanDisk SSD Plus and that was readable on my PC. OS is stock from what deskpi gave.

Any help at all would be great

**Comment by Aasikki** [comment]
@babyheysus OnePlus chargers only give 2A unless you are charging a oneplus devie as they don't support Qualcomm Quickcharge. Try another charger that supports QC.

**Comment by Mister-42** [comment]
> > Hello @yoyojacky @deskpi
> > Saying >refunded< on the order page does not make the money magically appear back on my PayPal account. Ignore emails also does not fix this. PROCEED WITH THE REFUND AT ONCE!
> 
> Reminder for @yoyojacky @deskpi. I'm gonna repeat this over and over until you have done as promised.

Still waiting @yoyojacky @deskpi

**Comment by simonduane** [comment]
> I bought one of the new spec cases about ten days ago (from a UK supplier) and find the my Pi now freezes once or twice a day with the white disk-activity light flashing furiously. Previously, it was absolutely stable when installed in a simple case with a fan and with the SSD drive connected via a StarTech USB32SSAT3CB (Chipset: ASM1153E) USB3 to SATA adapter. That chipset supports UASP and TRIM.
> 
> Removing the drive (Kingston 120 GB SA400S37/120G) from the DeskPi Pro case and reconnecting it externally with the adapter cable to a USB3 port has restored the stability that I enjoyed previously. Boot time is subjectively faster too.
> 
> There is clearly still something wrong with the revised board. I will be requesting a refund.

I can confirm this behaviour. New DeskPi delivered 1 week ago, assembled only today. Works, unless SSD is installed inside case and USB plug is fitted. Then no HDMI video on either port. It boots ok, because remote SSL connection is possible. But not VNC, which only says "Cannot currently show the desktop".
Strangely, rebooting without the USB plug brings up the desktop on a monitor (and via VNC), inserting the plug blanks the monitor, but the remote desktop via VNC survives. I suspect that by the time the HDMI signal has been routed out the back of the DeskPi case, it's too marginal to drive the monitor. I've seen inconsistencies a bit like this with pi-zero.
This is too flakey to be useful for me. I bought the case to run the pi as a desktop. With a monitor. At this price, it should just work. I'm sending it back.

**Comment by Aasikki** [comment]
@simonduane fuck that I'm going to ask for a refund now.

**Comment by G6EJD** [comment]
> > I bought one of the new spec cases about ten days ago (from a UK supplier) and find the my Pi now freezes once or twice a day with the white disk-activity light flashing furiously. Previously, it was absolutely stable when installed in a simple case with a fan and with the SSD drive connected via a StarTech USB32SSAT3CB (Chipset: ASM1153E) USB3 to SATA adapter. That chipset supports UASP and TRIM.
> > Removing the drive (Kingston 120 GB SA400S37/120G) from the DeskPi Pro case and reconnecting it externally with the adapter cable to a USB3 port has restored the stability that I enjoyed previously. Boot time is subjectively faster too.
> > There is clearly still something wrong with the revised board. I will be requesting a refund.
> 
> I can confirm this behaviour. New DeskPi delivered 1 week ago, assembled only today. Works, unless SSD is installed inside case and USB plug is fitted. Then no HDMI video on either port. It boots ok, because remote SSL connection is possible. But not VNC, which only says "Cannot currently show the desktop".
> Strangely, rebooting without the USB plug brings up the desktop on a monitor (and via VNC), inserting the plug blanks the monitor, but the remote desktop via VNC survives. I suspect that by the time the HDMI signal has been routed out the back of the DeskPi case, it's too marginal to drive the monitor. I've seen inconsistencies a bit like this with pi-zero.
> This is too flakey to be useful for me. I bought the case to run the pi as a desktop. With a monitor. At this price, it should just work. I'm sending it back.

VNC not able to display is nearly always because the RPi display resolution is not set correctly to say 1920x1080i if your monitor cant' display 60Hz signals.
My two cases are working correctly, I assume you used the new board supplied (two boxes arrived) to replace the driver chips, etc. I get good SSD performance too, not excellent but adequate:
pi@raspberrypi:~ $ sudo hdparm -Tt /dev/sda

/dev/sda:
 Timing cached reads:   1306 MB in  2.00 seconds = 653.14 MB/sec
 Timing buffered disk reads: 588 MB in  2.00 seconds = 294.0 MB/sec

The only issue I've found is the fan stops working the moment a USB device is plugged in and is allocated to USB0 also used for the fan control. I tried changing the USB address in the Python code, but no difference.

**Comment by simonduane** [comment]
> VNC not able to display is nearly always because the RPi display resolution is not set correctly to say 1920x1080i if your monitor cant' display 60Hz signals.

Nearly always, maybe, but not in this case. I was in an extended and very helpful exchange with RealVNC support last year over "Cannot currently show the desktop" messages - sometimes this message clears itself but only after a really long time (and sometimes never). Perhaps you didn't read my comment as far as the "... rebooting without the USB plug brings up the desktop on a monitor (and via VNC)." part? Meaning, it's not because of any settings that VNC isn't working.

All I can say is, your two cases work?
Fine. Mine doesn't.
And yes, of course I swapped out the old board for the new. I also fitted the RF interference-suppressing conductive cotton insert between the USB ports. If it looks like a bodge to get over flaky behaviour, maybe that's exactly what it is ... and maybe it doesn't always do the job.

**Comment by Mister-42** [comment]
> > > Hello @yoyojacky @deskpi
> > > Saying >refunded< on the order page does not make the money magically appear back on my PayPal account. Ignore emails also does not fix this. PROCEED WITH THE REFUND AT ONCE!
> > 
> > 
> > Reminder for @yoyojacky @deskpi. I'm gonna repeat this over and over until you have done as promised.
> 
> Still waiting @yoyojacky @deskpi

Email doesn't help. Posting in this issue doesn't help. This is very unprofessional.

**@yoyojacky @deskpi process that F@%$ing refund already!!!**

**Comment by leicray** [comment]
Update on my experience:

Previously, my Pi would crash once or twice a day with the white disk-activity light flashing furiously when installed in the DeskPi Pro case. I also posted my experience (also as leicray) on another forum hoping for guidance from there too: [TechForum Support](https://forum.digikey.com/t/deskpi-pro-bad-sata-controller/13490/14) It might be useful to read through that thread to understand the sequence of events and to see the experiences of other users.

The best advice from TechForum Support was to keep trying with the Pi in the DeskPi Pro case and wait for a crash and then take a look at the contents of the kernel log just before the crash: <code>/var/log/kern.log</code>

Before we get to that, it took nearly three days this time before it crashed again. Luckily, I had a a terminal window open on the desktop. However, some commands worked within the terminal, but others did not. For example I was able to <code>cat</code> the contents of kern.log but could not save it, as <code>nano</code> would not load and redirecting to output from <code>cat</code> to a file would not work either.

At this point, I could not connect to the pi using ssh or VNC. In addition, the desktop was frozen apart from the open terminal. It's probably best to say that the pi was only just alive at this point.

The content of kern.log shortly before the "crash" is:
<pre>
pi@raspberrypi:~ $ uptime
13:12:00 up 2 days, 21:16, 2 users, load average: 13.02, 10.45, 9.56
pi@raspberrypi:~ $ cat /var/log/kern.log
ay 3 11:46:09 raspberrypi kernel: [244220.207312] xhci_hcd 0000:01:00.0: WARN Cannot submit Set TR Deq Ptr
ay 3 11:46:09 raspberrypi kernel: [244220.207325] xhci_hcd 0000:01:00.0: A Set TR Deq Ptr command is pending.
ay 3 11:46:09 raspberrypi kernel: [244220.358262] usb 2-1: reset SuperSpeed Gen 1 USB device number 2 using xhci_hcd
pi@raspberrypi:~
</pre>

There are ten virtual bonus points on offer for anybody who can guess how I did capture the the content of kern.log and it was not by copying manually from the screen to a plain text file.

The good news is that the UK supplier (PiHut) has issued an RMA number and provided a pre-paid return label. In an ideal world, I would like to have a working case, but there is clearly something very wrong even with the revised board.

Any additional comments or encouragement would be most gratefully accepted.

**Comment by simonduane** [comment]
> Update on my experience:
...
> The good news is that the UK supplier (PiHut) has issued an RMA number and provided a pre-paid return label. In an ideal world, I would like to have a working case, but there is clearly something very wrong even with the revised board.
> 
> Any additional comments or encouragement would be most gratefully accepted.

Thanks for the update.
Well I'm encouraged anyway - I got my DeskPi from the same source. Fingers crossed they'll be similarly helpful to me. I went through enough time-consuming log-capturing games in trying to pin down the cause of my VNC problems last year. I'm really not up for more of that - I was really hoping for a nice tidy pi desktop system that just works (and am planning on trying the Argon ONE M.2 next)...

**Comment by nitz** [comment]
>  I was really hoping for a nice tidy pi desktop system that just works (and am planning on trying the Argon ONE M.2 next)...

I actually ordered the Argon One M.2 at the same time I ordered the DeskPi, so I've had a pi in it basically the whole time. It's my "daily driver" pi, and is honestly fantastic. The M.2 version has the case that does the micro hdmi to hdmi conversion too, which was a big deal for me and part of what made me go for it as well. But it does *fantastic* with thermals, is incredibly clever in it's design and machining, and still has a lot of the features I had hoped for. Sure no SATA, but if you're looking to go M.2 anyways, I don't think there's a better option.

**Comment by simonduane** [comment]
> > I was really hoping for a nice tidy pi desktop system that just works (and am planning on trying the Argon ONE M.2 next)...
> 
> I actually ordered the Argon One M.2 at the same time I ordered the DeskPi, so I've had a pi in it basically the whole time. It's my "daily driver" pi, and is honestly fantastic. The M.2 version has the case that does the micro hdmi to hdmi conversion too, which was a big deal for me and part of what made me go for it as well. But it does _fantastic_ with thermals, is incredibly clever in it's design and machining, and still has a lot of the features I had hoped for. Sure no SATA, but if you're looking to go M.2 anyways, I don't think there's a better option.

Yet more encouragement - cheers for that, it's just what I want to hear. (I only bought the SSD to go with the case, so SATA vs M.2 was an arbitrary choice for me ... I'm hoping that PH will look favourably on my request to swap SSDs too. It's not like it's even been formatted or anything yet ...)

**Comment by leicray** [comment]
I am slightly wary of going with the Argon One M.2 case for three reasons:

1. As far as I can see, the Argon case does not provide an accessible microSD slot: please correct me if I'm wrong.
2. I would now have to buy an M.2 drive to replace the SATA drive that I currently use in the DeskPi Pro case.
3. I cannot see how to write an image of the SATA drive to the M.2 drive without buying an M.2 to USB adapter.

On the plus side, it does have the ASmedia ASM1153E chipset which does provide UASP support, and presumably TRIM support too.

Any thoughts?

**Comment by nitz** [comment]
It doesn't have an easy to get ot SD slot, no. That was a non-issue for me since I intended to boot from the SSD, and attach any removable storage I need via USB. That said, It's just the base screws (4 of them) to remove the bottom which will let you get to the SD slot if you need to. The non-M.2 base does have access to it, so I imagine it's probably at least a concern for them if they ever do another revision.

Your point 2 is completely right. I was already using a M.2 SSD (and an adapter to a 2.5" SATA when I needed it,) so that's an easy move for me, but certainly more difficult for people in your case. That said, eBay is a great way to get close to if not exactly what you paid for if you flip your SSD there.

Your third point: you kinda get one automatically with the case: the base of the case/M.2 board is connected to the Pi via a USB 3 bridge the same way the DeskPi does, so you can absolutely use it to write your image to. I did it with mine directly from the Pi before I cased it up!

And for booting, I've only used two different M.2 drives in it so far, but neither of them had issues booting!

**Comment by mjeshurun** [comment]
> I am slightly wary of going with the Argon One M.2 case for three reasons:
> 
> 1. As far as I can see, the Argon case does not provide an accessible microSD slot: please correct me if I'm wrong.
> 2. I would now have to buy an M.2 drive to replace the SATA drive that I currently use in the DeskPi Pro case.
> 3. I cannot see how to write an image of the SATA drive to the M.2 drive without buying an M.2 to USB adapter.
> 
> On the plus side, it does have the ASmedia ASM1153E chipset which does provide UASP support, and presumably TRIM support too.
> 
> Any thoughts?

Based on this blog post and comments discussion, it appears the Aragon One M.2 currently doesn't supports TRIM.
[https://www.jeffgeerling.com/blog/2021/argon-one-m2-raspberry-pi-ssd-case-review](https://www.jeffgeerling.com/blog/2021/argon-one-m2-raspberry-pi-ssd-case-review)

**Comment by leicray** [comment]
Hi @nitz,

Thank you for the prompt reply!

> Your third point: you kinda get one automatically with the case: the base of the case/M.2 board is connected to the Pi via a USB 3 bridge the same way the DeskPi does, so you can absolutely use it to write your image to. I did it with mine directly from the Pi before I cased it up!

The fact that I can use the base of the case as a temporary adapter makes perfect sense. Kicking myself that I did not work that out for myself.

**Comment by leicray** [comment]
Hi @mjeshurun

> Based on this blog post and comments discussion, it appears the Aragon One M.2 currently doesn't supports TRIM.
https://www.jeffgeerling.com/blog/2021/argon-one-m2-raspberry-pi-ssd-case-review

I might hold off for a bit in the hope that the makers of the Argon case do update the firmware to support TRIM.

**Comment by jamsoft** [comment]
Thought I'd chip in here.

I just bought a DeskPi Pro, SSD and Pi 4B 8Gb last week from PiHut I had read about the lack of TRIM etc but this is really just a 'nix scratch pad play thing so totally not mission critical at all.

Built it all and installed Ubuntu 64 bit pi build. I'm typing this on it now. I have seen the crazy flashing USB light a number of times, requiring a power cycle to fix.

WiFi is dog slow and I've turned BT off for now as that seemed to be impacting WiFi ... but then that's also true of my stupid mac ... and it's not so magic mouse but that's another story.

The fan control install under Ubuntu 64 is pretty much broken. I've managed to get manual control working but really, what use is that? Cooling has to be controlled dynamically or it's "just a fan".  Someone else suggested in another ticket the drivers need recompiling for 64bit but I don't know where to start with that at the mo. Also, anything other than 100% fan speed has an annoying vibration resonance that I haven't identified and fixed yet.

Seems off manual control works but automatic doesn't.

All in all, case looks and feels great but there are definitely issues to be ironed out and my experience overall is definitely on the slightly frustrated side.

Currently eyeing up a NUC to be honest ...

**Comment by G6EJD** [comment]
Same experiences here, but I discovered I get good WiFi connections on 5 GHz and now the WiFi is fast. The metal case is not conducive to good RF links.
I have the same annoying buzz / resonance from the fan, not solved that yet.
When I plug in a USB device the fan stops, examination of the script reveals it uses usb0 which is also being assigned to the first device plugged in. I am going to allocate and name a USB port say usbfan and change the script to use that to solve the problem. The cpu runs at about 40° most of the time and the fan is off, so cooling is good. Overall a neat case with issues.

**Comment by jamsoft** [comment]
Forgot to mention. The power light always stays blue now I'm running Ubuntu. Would power down and turn red under Raspian so definitely issues there in the config script.

Currently trying to get GPU accelration working and failing at that but that's nothing to do with this case.

I'm not driven enough to get a refund but the experience is a bit lackluster for what is THE most expensive Pi case on the market ...

I'm a bit picky with my hardware builds so these issues are actually quite annoying on that level.

**Comment by jamsoft** [comment]
@G6EJD  Hmm ... 5Gz is pretty useless unless you're in the same room as the WiFi source.

**Comment by jamsoft** [comment]
On another issue here a chap posted this list of issues:

    Front USB not working (or not properly)
    USB dongle fail to use correctly the internal SSD/HDD
    WIFI disturbance
    Loading time takes too long.
    Fan speed control should be better implemented for easy modification on OS, example on Ubuntu 20.10

That's an issue WITH EVERY SINGLE FUNCTION OF THE CASE other than it being a case ... anyway, enough moaning.

**Comment by leicray** [comment]
I have just heard from The Pi Hut, here in the UK, that they have decided to suspend sale of the case until matters are resolved.

**Comment by simonduane** [comment]
@leicray interesting - I just sent mine back with its RMA yesterday.

**Comment by mjeshurun** [comment]
I sent Deskpi a refund request two days ago. 
Didn’t hear back from them yet, but I’ll give this another week, before contacting them again about it.

**Comment by risa2000** [comment]
FWIW, I emailed sales on April 20, asking for a refund and instructions how to proceed. They never replied, but sent me an advance notice about shipping the replacement board. So I went to PayPal customer protection and raised a complaint there while requesting a full refund.

**Comment by G6EJD** [comment]
I’m wondering what’s different with my two cases which apart from the fan control using USB0 which I fixed by allocating the port to the  fan and changing the Python script all is working as expected. I’ve since found if I take the RPi out of the case it won’t connect to a 2.4G access point, well unless I force it to do so via wpa_supplicant. 5G range is a good 6M but I’m using a wifi6 mesh. I note the new boards have a screening can over components directly below the Pi USB ports. In one case I’m using an M.2 drive the other a 2.5” SSD both boot in identical times of about 7.5secs according to system-analyse and I’m getting near maximum r/w speeds for the two drives. Processor temp is a steady 39°c at most 46°. There are matching serial number stickers on the top and bottom pcbs the originals were blank. I think the version number is 3.1 on the new boards. The fan buzz is of very low sound level, but doesn’t other me as at 1M it’s inaudible.

**Comment by mjeshurun** [comment]
An automated reply I got regarding my refund request email from Deskpi’s sales account.
Let’s see if they answer in the next few days. 

```
Hello,

Thanks for your letter.

Please
kindly note: we will have Labor Day holiday during May 1st to 5th.

We will reply you when we back to the office on May 6th.

Have a nice day!
```

**Comment by jamsoft** [comment]
@G6EJD I've only got one, bought just days before PiHut decided to stop selling them.

I can honestly report the same. The fan and power buttons have issues under Ubuntu 21.04 but all in all it *is* working for me also.

From cold boot to entering my password is around 28 seconds and it takes maybe another 10-15 to get into the desktop proper. I feel that's actually pretty acceptable performance wise.

It's using a 2.5in WD Green also bought from PiHut.

I do think they're right to suspend selling it though. Too many people *are* having showstopper problems.

**Comment by mjeshurun** [comment]
Just to update, I’ve received my refund. 
I also got confirmation from PayPal.

**Comment by Mister-42** [comment]
> Just to update, I’ve received my refund.
> I also got confirmation from PayPal.

I'm still waiting. @yoyojacky @deskpi process that refund!

**Comment by haan2787** [comment]
i wil not sent it back for a refund ,i want the new pcb board like deskpi promist
but digi-key  where i have this deskpi from wil not help they say "you must call deskpi"
so i ask deskpi for sending me the newest pcb boad ,and if i have to pay ,i like to hear
made this reguest 2 days ago till now no aswer
if someone else has better results then i like to know

**Comment by G6EJD** [comment]
I have the new pcb fitted but have today decided to return mine for a refund. Reliability is poor, let alone the on-going WiFi, USB, fan, ssd access issues. I decided to run the RPi + SSD outside the case and everything works as expected, WiFi can connect to either 2.4GHz or 5GHz, USB ports work, SSD access is trouble free and no random crashing. I’ve ordered an Argon one m2 to try that.

**Comment by leicray** [comment]
@G6EJD

> I decided to run the RPi + SSD outside the case and everything works as expected.

That is **exactly** the same as my experience. I was beginning to think that every user had similar but non-identical experiences. I'm holding off returning the case to Pi Hut just yet as they do seem to be investigating the situation.

**Comment by G6EJD** [comment]
When I saw the case I thought the ventilation holes would provide a sufficient path for the RF coupled with the Perspex end-plates, but unfortunately the case provides an effective shielded enclosure and for that aspect there is no solution. 
The fan/usb port can be solved. Assign a logical name to USB0, I called it USBFAN and then modified the Python script so it was always in use when other devices were plugged in.
The SSD driver speed didn’t bother me as I was getting near maximum transfer speeds for the SSD (550MB/s) I’m using, but unexplained crashing after a few days on my webserver was irritating and doesn’t happen out of the case with an SSD adapter.
So yes they can probably resolve the issues, but not the WiFi unless they cut out a reasonably large part of the case above the WiFi chip and maybe replace with a plastic insert. 

I’ll get my Argon One M.2 tomorrow and I’ll be keen to compare and contrast, the upside is there are few reported issues with it in the V2 variant. 

If the issues are fixed I’ll purchase one again. I got mine from PiHut and they didn’t hesitate to issue an RMA seeking a return, so I did. They have withdrawn sales for the time being.

Good luck with yours.

**Comment by G6EJD** [comment]
New  Argon M2 working perfectly, cpu temp about 45-degs, slightly faster than the DeskPi Pro, so far pleased. No Wi-Fi issues to report,, both 2.4G and 5G can be received. Time to boot 11.63-secs according to systemd-analyze.

Raspberry Pi Diagnostics - version 0.9
Fri May  7 12:20:48 2021

Test : SD Card Speed Test
Run 1
prepare-file;0;0;321254;627
seq-write;0;0;306242;598
rand-4k-write;0;0;53939;13484
rand-4k-read;53894;13473;0;0
Sequential write speed 306242 KB/sec (target 10000) - PASS
Random write speed 13484 IOPS (target 500) - PASS
Random read speed 13473 IOPS (target 1500) - PASS
Test PASS

**Comment by haan2787** [comment]
no answer from digi-key and deskpi  
bad support shame on them
i am going to build a case with the help of geekpi
using the cooler and
the sabre ssd to usb incloser
thanks for the help

**Comment by simonduane** [comment]
full marks to Pi Hut - no quibble, the refund for the case and SSD that I returned, arrived yesterday. Pi users in the UK probably don't need my recommendation, but I'll make it anyway. Very strongly.

**Comment by mjeshurun** [comment]
Although I already received my refund, but I just got this email from Deskpi's sales team, regarding the future support of UASP and TRIM in a third version of Deskpi Pro.

```
Hi friend,
Thanks for your time, sorry for my delay because we had Labor Day holiday in the past few days.

It is delayed, we delayed the launch and delivery of new products. in order to meet the needs of most consumers, we want to update DeskPi Pro again to make it also supports UASP / TRIM.

Our tech team is testing the new board (with another new chip) if it can support UASP / TRIM. We may get the testing result recently.

We can also full refund you if you want. Please kindly let me know your idea.

Sorry for any inconvenience! Thank you!

Best regards,
DeskPi Team
```

**Comment by G6EJD** [comment]
You should also find a solution for the USB port being reassigned to e.g. a keyboard when plugged in as it stops the fan from working and improve the Wi-Fi signal by removing some case material over/around the RPi Wi-Fi chip.

Also the front USB ports can stop working for some reason.

The case is the best I've seen but unusable in its current form.

**Comment by leicray** [comment]
> Also the front USB ports can stop working for some reason.

The behaviour of the front USB ports is definitely odd. For me, they work fine if I boot from the SSD, but they do not work if I boot from a microSD card. BTW, I'm running Raspberry Pi OS 5.10.

**Comment by haan2787** [comment]
o thats good news if they make a new board 
are they helping everyone who bought a deskpi for example digi-key
they can better stop selling until everything is working 
and contact copany like digi-key pihut etc to give everyone a good solution 
i have bought a other case ,but i want the dekpi working
cost me a lot of money now
problems : slow speeds with usb now hdmi output connecting to kvmswitch (tesmart 4x4 4k) hdmi only works directly connecting to the monitor ,usb 2.0 connection no booting ,booting ssd inside does not work sometimes slowly ,external ssd slow ,external hhd slowly  so a lot of things 
i hope that deskpi wil help crossing my vingers for that

**Comment by leicray** [comment]
> and contact copany like digi-key pihut etc to give everyone a good solution
 
Pi Hut has contacted me to say that 52Pi (the manufacturer) says that there be "some faulty units". Pi Hut say that think that the problem is widespread.

> now hdmi output connecting to kvmswitch (tesmart 4x4 4k) hdmi only works directly connecting to the monitor

That's an interesting point. I too had been using a KVM switch **before** I transferred the Pi to the DeskPi case, and it worked as expected. However, it suddenly stopped working around the time of a Windows 10 update that was claimed to increase HDMI security. I never did resolve the problem and instead now use VNC to connect to the Pi desktop.

**Comment by G6EJD** [comment]
Looking at the PCB design / layout there appears to be just a simple microHDMI pin-to-pin tracking between micro and full size HDMI connectors, making it difficult to attribute the switching problem to the Deskpi case/pcbs, unless the same arrangement with the bare Pi using microHDMI works without issues.

**Comment by jamsoft** [comment]
I've gone back to [Raspios OS 64bit](https://downloads.raspberrypi.org/raspios_arm64/images/) since Ubuntu was just throwing up too many brick walls for my use case. Key development tools that are 3 major versions behind my main environment tools isn't inspiring.

However, the DeskPi now seems to be working ...

I'm booting from SSD, the auto fan control is working. It really does need [ramp up and down behaviour](https://github.com/DeskPi-Team/deskpi/issues/40) rather than just plain on / off as the CPU fluctuates around the first threshold, but it is working. The power button also seems to be behaving as expected as well.

I've just run some speed tests on the Pi, inside the case. My [router](https://www.asus.com/uk/Networking-IoT-Servers/WiFi-6/All-series/RT-AX92U/) is two floors below me, so quite a distance. I have a mixture of Cat6 wired and wireless devices in this room that still have really good connections so it certainly *could* get a solid connection if working correctly, which it seems to be. But then I've only really used the Pi WiFi from inside the case so I've no baseline expectation at the mo.

### WIFI

```
speedtest-cli
Retrieving speedtest.net configuration...
Retrieving speedtest.net server list...
Selecting best server based on ping...
Hosted by Fasthosts Internet (Gloucester) [59.89 km]: 29.494 ms
Testing download speed................................................................................
Download: 9.39 Mbit/s
Testing upload speed................................................................................................
Upload: 9.94 Mbit/s
```
### SSD

**Write Speeds**
```
dd if=/dev/zero of=./speedTestFile bs=20M count=5 oflag=direct
5+0 records in
5+0 records out
104857600 bytes (105 MB, 100 MiB) copied, 0.695178 s, 151 MB/s
```
**Read Speeds**
```
dd if=./speedTestFile of=/dev/zero bs=20M count=5 oflag=dsync
5+0 records in
5+0 records out
104857600 bytes (105 MB, 100 MiB) copied, 0.569607 s, 184 MB/s
```

I also took those terminal snippets and copied and pasted them into a file on a thumb drive plugged into the front USB ports in order to test them and get the text to paste in here ... #ludditemode lol

All in all, it looks like my unit is pretty functional.

**Comment by haan2787** [comment]
thanks for the test
i have the raspberry without a case running and normal hdmi and the espected speed with the ssd and hdd
the router acts normal  so no problems
just like i want to see  ,the problem is the deskpi
i realy hope for a solution but in the mean time a wait for the geekpi

**Comment by leicray** [comment]
> I've gone back to [Raspios OS 64bit](https://downloads.raspberrypi.org/raspios_arm64/images/)

My understanding is that the 64-bit version is still in development. Please correct me if I'm wrong. I'm using the 32-bit version as I only have 4 GB RAM and the 64-bit version would yield no advantages.

> However, the DeskPi now seems to be working ...

Weirdly, so is mine. It has been up for nearly five days now, whereas it had previously been crashing once or twice a day. The power button works exactly as expected.

BREAKING NEWS: it has just crashed again in the last couple of minutes!

Still no further forward I fear.

**Comment by G6EJD** [comment]
Crashing after about 5-days, that was my experience too, since I changed to the Argon One M2 no crashing, no WiFi drop-outs or USB problems on my webserver.

**Comment by leicray** [comment]
> > My understanding is that the 64-bit version is still in development. Please correct me if I'm wrong. I'm using the 32-bit version as I only have 4 GB RAM and the 64-bit version would yield no advantages.
> 
> Strictly speaking a 64 bit chip really should be running a 64 bit OS. It effects things like addressable RAM etc so my having 8GB 64bit makes a stack of sense.
> 
> Besides, isn't all of Linux constantly "in development" ;)

I was perhaps not as clear as I might have been. If I run Raspberry Pi Imager, it only offers 32-bit versions of Raspberry Pi OS. I interpret that to mean that the 64-bit versions are still not far enough along the development path to be considered ready for general release.

As I said previously, my Pi has just 4 GB RAM. The value of the 64-bit version would only be realised if I had the 8-GB version.

**Comment by jamsoft** [comment]
No you were clear, just missed the joke.

You should also see better performance too. I'm not as familiar with ARM to be honest but I know, for example, x64 has additional instructions too not just greater address space.

**Comment by yoyojacky** [comment]
> 您比我幸运的多，因为我发了3封邮件，没有一个答案...
sorry for my slow response, we will send the replacement one by one.. Because of the epidemic situation and the current lack of chips, the manufacturing time is relatively long, and we have also strengthened quality control. In order to provide better products, we have arranged many tests, so the speed will be slower. Please rest assured that we will provide your deskpi Provide solutions to problems that arise.

**Comment by Mister-42** [comment]
> > 您比我幸运的多，因为我发了3封邮件，没有一个答案...
> > sorry for my slow response, we will send the replacement one by one.. Because of the epidemic situation and the current lack of chips, the manufacturing time is relatively long, and we have also strengthened quality control. In order to provide better products, we have arranged many tests, so the speed will be slower. Please rest assured that we will provide your deskpi Provide solutions to problems that arise.

What about my cancelled order and the lack of refunding?

**Comment by yoyojacky** [comment]
> 是的，该网站已从“三月中旬”静默更新为“三月末之后”，而没有任何通知或礼节性电子邮件。海事组织，这极大地扰乱了商业惯例。几个月来稀疏，中断或根本不存在的通信，然后因农历新年或其他假期而关闭，这是非常可悲的。3个月的评论和一封电子邮件，以修复PCB板。对于应该在开发之前就发现的问题，因为该芯片在SBC方面存在众所周知的问题。
> 不过我知道了，他们希望在m.2氩气之前或通常更快的速度淘汰它，所以事情可能会发生。我得到它。
> 但是仍然对发生的事情，任何类型的eta都保持零沟通，甚至是“嘿！我们还在这里！努力工作”。那才是我最大的收获。（每周一次？每月一次）发布更新不到5分钟即可改善客户关系。但不是？我们不应该得到吗？还是要求太多？您是否在努力工作_，_无法向我们提供最新消息？
> 是的，我希望在三月中旬前得到一个固定的PCB板，所以我大概需要一两个星期的时间才能退款，如果这样不起作用，则会因我购买产品/服务而产生欺诈性收费，并且未完全提供商定的产品。
> 看起来我有点咸，但是从字面上看，不到5分钟即可发布更新。我们都有手机和互联网连接。这样就告诉我他们很少在乎
It’s not that we don’t care, we care a lot about any customer who has purchased our products. We hope to provide better service, but we need to constantly test and try to find the best combination. It will take some time. Please don’t lose patience. We release Updates are responsible for customers, so customers should not be harassed frequently. We are indeed lacking in communication, we will work hard to improve, and we are very sorry for the inconvenience caused to you.

**Comment by yoyojacky** [comment]
> @ Pi-Mania，很幸运，Deskpi的销售人员停止了对我的回应，因为他们显然没有按3月中旬或下旬的截止日期。
> 而且，如jojobrogess所言，他们网站上的更新将花费很少的时间，并且会给人留下更成熟的印象。
> 我只是希望新的PCB确实能解决所有问题。...
> 我的矿井刚开发出一个新的：欠压警告和未重启的重新启动。:-(

Could you please send me the environment of your Raspberry Pi ? I do not meet that problem during test... I need more information please...

**Comment by yoyojacky** [comment]
> 3Gb芯片（174c：1153）而不是6Gb（174c：55aa

some chip with :174c：1153 does not support UASP and TRIM but supports USB booting well.
chip with 174c：55aa supports UASP and TRIM and supports USB booting but need update firmware, we are going to pack a software pack for all of you so that you can upgrade the firmware by yourself and enable TRIM function.

**Comment by yoyojacky** [comment]
> digi-key和deskpi没有答案
> 不好的支持，让
> 我感到羞耻。我要在geekpi的帮助下，
> 使用冷却器和
> saber ssd向usb内藏器制作一个盒子，
> 感谢您的帮助

what can i do for you sir? do you have any problem with installing the ice cooler and ssd ?

**Comment by Mister-42** [comment]
> > 那我被取消的订单又缺少退款怎么办？
> > Could you please send your order number to [sale@deskpi.com](mailto:sale@deskpi.com) ？ it may take a while, just be patiant ...

I have send multiple emails already, without resolution!

**Comment by yoyojacky** [comment]
> ，

OK, could you please tell me your order number and i will follow up your case.

**Comment by Mister-42** [comment]
> > ，
> 
> OK, could you please tell me your order number and i will follow up your case.

#DP1747

**Comment by yoyojacky** [comment]
> > 同样，前端USB端口由于某些原因可能会停止工作。
> 
> 前面的USB端口的行为肯定很奇怪。对我来说，如果我从SSD引导，它们可以正常工作，但是，如果我从microSD卡引导，它们将无法工作。顺便说一句，我正在运行Raspberry Pi OS 5.10。

did you add : dtoverlay=dwc,dr_mode=host to config.txt file on you MicroSD card ?

**Comment by yoyojacky** [comment]
> Crashing after about 5-days, that was my experience too, since I changed to the Argon One M2 no crashing, no WiFi drop-outs or USB problems on my webserver.

Crashing ? what kind of crashing ?  kernel panic ? or any information about your OS please ?

**Comment by yoyojacky** [comment]
> > > ，
> > 
> > 
> > OK, could you please tell me your order number and i will follow up your case.
> 
> #DP1747

Got it, I am contacting with sales dept right now. thank you.

**Comment by jamsoft** [comment]
I've just discovered that you can't plug a standard ribbon cable onto the GPIO breakout on the DeskPi daughterboard ... despite there even being a cut out on the rear acrylic panel ...

Has anyone managed too?

**Comment by jamsoft** [comment]
Took off the rear panel, managed to get it squeezed on, did that a few times, replaced the panel and it now seems a little happier.

Maybe revisit this bit of the design as the tolerances are really, really tight.

**Comment by leicray** [comment]
> > > 同样，前端USB端口由于某些原因可能会停止工作。
> > 
> > 
> > 前面的USB端口的行为肯定很奇怪。对我来说，如果我从SSD引导，它们可以正常工作，但是，如果我从microSD卡引导，它们将无法工作。顺便说一句，我正在运行Raspberry Pi OS 5.10。
> 
> did you add : dtoverlay=dwc,dr_mode=host to config.txt file on you MicroSD card ?

I believe that this question relates to one of my posts. It's hard to say for certain as I have absolutely no knowledge of Chinese. Could we please keep the postings in English?

The answer to the question, is no. Normally I boot from the SSD.

**Comment by haan2787** [comment]
yoyojacky  thanks for your reaction 
i bought 1 deskpi-pro  from digi-key
i askt them if they no about the not working pcb (hdmi-usb-ssd)
they told to contact you deskpi 
i want to know if i olso getting the replacement pcb board (new board)
now its trash because i can not use it 
and i was waiting from october2020 for this case 
now  11-5-2021 i have trash
i realy hope you can help me

**Comment by yoyojacky** [comment]
> > > ，
> > 
> > 
> > OK, could you please tell me your order number and i will follow up your case.
> 
> #DP1747

Your order on the PayPal interface shows that there is no replacement button after it is completed. The platform has changed on 4.21. The refund has been refunded a long time ago. Please check the following PayPal account information， 
![QQ图片20210512111220](https://user-images.githubusercontent.com/6893075/117912906-f0764c80-b312-11eb-9c2a-96c4995c1040.png)

**Comment by yoyojacky** [comment]
> I've just discovered that you can't plug a standard ribbon cable onto the GPIO breakout on the DeskPi daughterboard ... despite there even being a cut out on the rear acrylic panel ...
> 
> There doesn't seem to be enough clearance between the first row of pins and the PCB.
> 
> Has anyone managed too?

Can you provide photos of the installation? The GPIO cable should be passed through the gap of the USB interface

**Comment by yoyojacky** [comment]
> Took off the rear panel, managed to get it squeezed on, did that a few times, replaced the panel and it now seems a little happier.
> 
> Maybe revisit this bit of the design as the tolerances are really, really tight.

Are you saying that the gap between the acrylic shell and the aluminum alloy shell is too small?

**Comment by yoyojacky** [comment]
> please

That's just your post which i quote here,  I answer you question that you said the front usb port work well when you use SSD booting up raspberry pi but out of working once you use MicroSD card booting up,  it caused by you just did not add : `dtoverlay=dwc,dr_mode=host`  to config.txt file on you MicroSD card ?

**Comment by yoyojacky** [comment]
> yoyojacky thanks for your reaction
> i bought 1 deskpi-pro from digi-key
> i askt them if they no about the not working pcb (hdmi-usb-ssd)
> they told to contact you deskpi
> i want to know if i olso getting the replacement pcb board (new board)
> now its trash because i can not use it
> and i was waiting from october2020 for this case
> now 11-5-2021 i have trash
> i realy hope you can help me

OK, could you please send your order number and the issue of your board to sale@deskpi.com and cc to jacky.li@52pi.com

**Comment by Mister-42** [comment]
> > > > ，
> > > 
> > > 
> > > OK, could you please tell me your order number and i will follow up your case.
> > 
> > 
> > #DP1747
> 
> Your order on the PayPal interface shows that there is no replacement button after it is completed. The platform has changed on 4.21. The refund has been refunded a long time ago. Please check the following PayPal account information，
> ![QQ图片20210512111220](https://user-images.githubusercontent.com/6893075/117912906-f0764c80-b312-11eb-9c2a-96c4995c1040.png)

As I have tried to explain multiple times: this is not the case!
My best guess is that the cancellation has beendone after 6 months of payment, so a normal refund is not possible or visibe or whatever. But in any case, no money have been returned. You can check the Paypal account to verify my claim!

**Comment by jamsoft** [comment]
> 
> 
> > Took off the rear panel, managed to get it squeezed on, did that a few times, replaced the panel and it now seems a little happier.
> > Maybe revisit this bit of the design as the tolerances are really, really tight.
> 
> Are you saying that the gap between the acrylic shell and the aluminum alloy shell is too small?

No, the row of pins nearest the PCB have an incredibly tight tolerance making it hard to plug in a standard ribbon cable.

It's nothing to do with the alu case or acrylic panels.

**Comment by bpicon** [comment]
> some chip with :174c：1153 does not support UASP and TRIM but supports USB booting well.
chip with 174c：55aa supports UASP and TRIM and supports USB booting but need update firmware, we are going to pack a software pack for all of you so that you can upgrade the firmware by yourself and enable TRIM function.

@yoyojacky , From your comment I must understand that new boards are being shipped with 174c: 55aa and for these a firmware update is going to be enabled?
I have received 174c: 1153 (not 1153E), uasp and trim support is not available, the system directly enables the usb-storage driver. However, the manufacturer includes within the chip specifications  the uasp support (https://www.asmedia.com.tw/product/995Yq54sX2YI1th2/d1Eyq85QN8GhBwRC)
Can you tell me if the firmware update you propose will enable these features in my case, changing the product_id to 147c: 55aa?
Or that new boards are provided with different types of chip and I have gotten one that only allows usb booting?

Thanks.

**Comment by haan2787** [comment]
yoyojacky
I did sent you a email but sale@deskpi.com gives a error back
i hope that the mail is in youre emailbox
thanks

**Comment by leicray** [comment]
> I answer you question that you said the front usb port work well when you use SSD booting up raspberry pi but out of working once you use MicroSD card booting up, it caused by you just did not add : `dtoverlay=dwc,dr_mode=host` to config.txt file on you MicroSD card ?

I did not add the `dtoverlay=dwc,dr_mode=host` to the config.txt file on the microSD card. I think that the need to modify the file corresponds to the fifth **Fault** on the FAQs card in the box: "**Fault:** Abnormal USB interface on the front panel." This issue is not explained clearly enough as the problem and cure **only** relate to the situation where the Pi has been booted from the microSD card rather than from the SSD.

Clearer instructions would help too. Above, you have said that it is necessary to add`dtoverlay=dwc,dr_mode=host` to the config.txt file, but the instruction card says to add `dtoverlay=dwc, dr_mode=host` with a space after the comma. I think that including the space is incorrect.

**Comment by haan2787** [comment]
yoyojacky
i saw youre website  and many intresting thing on it 
but can you transform it in englisch so we al understand it (mabe a idee)
I hope everything is oke because we do not here from you for 4 days
please give us a word back
thanks

**Comment by leicray** [comment]
I posted my case back to Pi Hut (UK vendor) on Friday morning and it was delivered this morning. My refund, via PayPal, was sorted out within a couple of hours. Fantastic service!

**Comment by leicray** [comment]
> It seems that the firmware update is already official (the date April 20 must be an error, it will be May 20)
> If someone has tried updating, they could share their impressions.

Although its encouraging to see that a firmware update for UASP and TRIM is now available, I'm not convinced that it will necessarily cure the problem. My Pi is now back in its original simple case with the SSD connected via a StarTech adapter cable that supports both UASP and TRIM. It has been up for nearly eight days without problems.

If I was to substitute the SSD adapter with another that did not support UASP and TRIM, I doubt that it would cause the Pi to crash or kernel panic. Apart from anything else, the crashes/panics were happening when the device was idling. There must be something else going on in addition.

**Comment by csmale** [comment]
I received a replacement board on 20 April but it has the 174c:1153 chip. As I understand it I cannot use the firmware update because that is for the 174c:55aa hardware on the V3 board. Do I need a V3 board? How can I get one?

**Comment by bpicon** [comment]
> I received a replacement board on 20 April but it has the 174c:1153 chip. As I understand it I cannot use the firmware update because that is for the 174c:55aa hardware on the V3 board. Do I need a V3 board? How can I get one?

I received the board with the same chip (174c: 1153) and I have been able to update the firmware without problems.
The original version that it showed, before updating was 130531210000.
The update changes the product-id of the chip to 55aa and it is now detected by the kernel as compatible with the uas driver. Later I activated TRIM without problems and the performance data reported by agnostics has improved substantially.

pi@raspberrypi:~ $ lsusb
Bus 003 Device 003: ID 1a86:7523 QinHeng Electronics HL-340 USB-Serial adapter
Bus 003 Device 002: ID 05e3:0610 Genesys Logic, Inc. 4-port hub
Bus 003 Device 001: ID 1d6b:0002 Linux Foundation 2.0 root hub
**Bus 002 Device 002: ID 174c:55aa ASMedia Technology Inc. Name: ASM1051E SATA 6Gb/s bridge, ASM1053E SATA 6Gb/s bridge, ASM1153 SATA 3Gb/s bridge, ASM1153E SATA 6Gb/s bridge**
Bus 002 Device 001: ID 1d6b:0003 Linux Foundation 3.0 root hub
...

pi@raspberrypi:~ $ lsusb -t
/:  Bus 03.Port 1: Dev 1, Class=root_hub, Driver=dwc2/1p, 480M
    |__ Port 1: Dev 2, If 0, Class=Hub, Driver=hub/4p, 480M
        |__ Port 1: Dev 3, If 0, Class=Vendor Specific Class, Driver=ch341, 12M
/:  Bus 02.Port 1: Dev 1, Class=root_hub, Driver=xhci_hcd/4p, 5000M
    |__ **Port 1: Dev 2, If 0, Class=Mass Storage, Driver=uas, 5000M**
...

Raspberry Pi Diagnostics - version (unknown)
Sun May 23 14:14:33 2021

Raspberry Pi Diagnostics - version (unknown)
Sun May 23 18:08:50 2021

Test : SD Card Speed Test
Run 1
prepare-file;0;0;322837;630
seq-write;0;0;321254;627
rand-4k-write;0;0;48473;12118
rand-4k-read;50257;12564;0;0
Sequential write speed 321254 KB/sec (target 10000) - PASS
Random write speed 12118 IOPS (target 500) - PASS
Random read speed 12564 IOPS (target 1500) - PASS
Test PASS

Before update firmware:
![Deskpi_BeforeFlash](https://user-images.githubusercontent.com/76492726/119267879-bea59580-bbf0-11eb-9d60-fc19dd35dec5.PNG)

After update:
![Deskpi_AfterFlash](https://user-images.githubusercontent.com/76492726/119267894-d1b86580-bbf0-11eb-9bc4-0aab500a5c3b.PNG)

**Comment by csmale** [comment]
Thanks @bpicon - I went ahead and did the update. However I can no longer boot from the SSD - it does the pre-boot, then hangs for a couple of minutes and eventually dumps me into the recovery shell (initramfs). I can't "see" or access /dev/sda* and the white LED is on continuously (although it does flash during the earlier stages).

If I boot from an SD card I can mount the SSD without problems - it is detected as :55aa and is using uas.

So it looks like the hardware is OK but there is something in the boot sequence screwing it up. Does anyone have any ideas?

I am running Ubuntu 20.04 by the way.

**Comment by G6EJD** [comment]
In the boot config you will find that it is referencing the wrong device now, can’t find it…
Edit that and normal service will be resumed, use lsusb to find the new device ID

**Comment by csmale** [comment]
> In the boot config you will find that it is referencing the wrong device now, can’t find it…
> Edit that and normal service will be resumed, use lsusb to find the new device ID

Thanks for the suggestion... I have looked in the boot dir through all the config.txt, cmdline.txt etc files but I can't see any reference to either the old or the new device. Can you explain a bit more about what you mean?

**Comment by G6EJD** [comment]
What does this report as the boot device:
lsblk /dev/sda
do you see the SSD interface?

Run the command “gksu gedit” (or use nano or vi).
Edit the file /etc/fstab. Change the UUID or device entry with the mount point / (the root partition) to your new drive. ... 
Edit the file /boot/grub/menu. lst.

**Comment by mjeshurun** [comment]
Hello friends,
Has anyone received a completely new V3 Deskpi Pro order (not a pcb board replacement for an old order)? 
Are new V3 orders preconfigured to use USAP and TRIM out of the box, or users still need to manually run the process described on the Deskpi website? https://deskpi.com/blogs/news/about-uasp-trim
Also, on V3 which pcb board is used for the SSD, ASM1153 SATA 3Gb, or ASM1153E SATA 6Gb?

**Comment by haan2787** [comment]
no iam still waiting

**Comment by TeoTN** [comment]
I've received the new board but it's pretty much unusable, it keeps randomly crashing - I wasn't able to find any meaningful piece of log but it seems it attempts to reboot and fails. It seems like if SSD was disconnecting from time to time. The previous board didn't have that particular problem.
While DeskPi Pro looks nice, I regret buying it instead of Argon One, that was quite a waste of money...

**Comment by sjb5** [comment]
I received the new board.  Everything appears to be working as intended except for booting from SSD.  I performed the firmware update.  When I run lsusb, I get the following: Bus 002 Device 002: ID 174c:55aa ASMedia Technology Inc. Name: ASM1051E SATA 6Gb/s bridge, ASM1053E SATA 6Gb/s bridge, ASM1153 SATA 3Gb/s bridge, ASM1153E SATA 6Gb/s bridge.  My issue is similar to the issue described by csmale.  I cannot access the SSD when it is connected to the DeskPi Pro board.  The white LED turns on, but is on continuously.  It shows no activity.  If I connect the SSD by way of a Startech  SATA to USB cable, the system boots without a problem.  I have tried various old and new SSDs from Crucial, WD, Samsung and even Intel with the same result.  I have tried the current versions of both Raspbian and Ubuntu MATE.  The results are the same.  Unless I am missing something, I believe my new board is defective.

**Comment by csmale** [comment]
> I received the new board. Everything appears to be working as intended except for booting from SSD. I performed the firmware update. When I run lsusb, I get the following: Bus 002 Device 002: ID 174c:55aa ASMedia Technology Inc. Name: ASM1051E SATA 6Gb/s bridge, ASM1053E SATA 6Gb/s bridge, ASM1153 SATA 3Gb/s bridge, ASM1153E SATA 6Gb/s bridge. My issue is similar to the issue described by csmale. I cannot access the SSD when it is connected to the DeskPi Pro board. The white LED turns on, but is on continuously. It shows no activity. If I connect the SSD by way of a Startech SATA to USB cable, the system boots without a problem. I have tried various old and new SSDs from Crucial, WD, Samsung and even Intel with the same result. I have tried the current versions of both Raspbian and Ubuntu MATE. The results are the same. Unless I am missing something, I believe my new board is defective.

I got it going eventually, but it took a lot of hassle. I even managed to screw up my boot partition with all the desperate attempts to get it going. In the end I created a clean install of Ubuntu 20.04 LTS on an SD card, booted from that with the SSD connected, mounted the SSD partitions, and copied the boot partition and the firmware from the card to the SSD. After checking the config.txt was going to find the root partition on the SSD I could get it to start. I also remember starting from Raspbian and using raspi-config to set the boot order to start from the SD if present, otherwise SSD.
Now it runs OK - SSD performance is up to 700MB/sec - but it has spontaneously restarted once and the WiFi is not working properly. I have read there can be some RF interference between the USB3.0/SSD circuitry and the WiFi.

**Comment by sjb5** [comment]
csmale, thanks for the quick response.  My problem is that when I boot from the SD card with the SSD connected, I am unable to mount the SSD partitions.  When I run gparted, the SD card is the only device listed.  When I run fdisk -l, the SSD is not listed.

**Comment by csmale** [comment]
Aah, sorry, I misread a bit. When I had that problem in the beginning, it turned out that the four power pins from the main board had missed their correct holes in the daughter board. But you are probably not as clumsy as me...

It's definitely possible that your board is dead. I thought that several times during the most stressful 48 hours of my recent life.... And it still ended up being a software/config problem in my case.

**Comment by yoyojacky** [comment]
> > > 取下后面板，设法将其挤压，做了几次，更换了面板，现在看起来更快乐了。
> > > 也许重新审视设计的这一点，因为公差真的非常非常紧。
> > 
> > 
> > 你是说亚克力外壳和铝合金外壳的间隙太小了？
> 
> 不，最靠近 PCB 的那排针脚公差非常小，因此很难插入标准的带状电缆。
> 
> 与铝壳或亚克力板无关。



> I received the new board. Everything appears to be working as intended except for booting from SSD. I performed the firmware update. When I run lsusb, I get the following: Bus 002 Device 002: ID 174c:55aa ASMedia Technology Inc. Name: ASM1051E SATA 6Gb/s bridge, ASM1053E SATA 6Gb/s bridge, ASM1153 SATA 3Gb/s bridge, ASM1153E SATA 6Gb/s bridge. My issue is similar to the issue described by csmale. I cannot access the SSD when it is connected to the DeskPi Pro board. The white LED turns on, but is on continuously. It shows no activity. If I connect the SSD by way of a Startech SATA to USB cable, the system boots without a problem. I have tried various old and new SSDs from Crucial, WD, Samsung and even Intel with the same result. I have tried the current versions of both Raspbian and Ubuntu MATE. The results are the same. Unless I am missing something, I believe my new board is defective.

Could you describe what steps did you make ? to boot from SSD ?

**Comment by yoyojacky** [comment]
> I've received the new board but it's pretty much unusable, it keeps randomly crashing - I wasn't able to find any meaningful piece of log but it seems it ends up in rebooting spontaneously and freezes, probably same issue as mentioned earlier in this thread. It seems like if SSD was disconnecting from time to time. The previous board didn't have that particular problem.
> While DeskPi Pro looks nice, I regret buying it instead of Argon One, that was quite a waste of money...

what you mean `random crashing`  is ?  Does the SSD offline randomly?

**Comment by yoyojacky** [comment]
> csmale, thanks for the quick response. My problem is that when I boot from the SD card with the SSD connected, I am unable to mount the SSD partitions. When I run gparted, the SD card is the only device listed. When I run fdisk -l, the SSD is not listed. When I run lsblk, the SD card is listed, but not the SSD.

please try to add `sudo` before command and try again...

**Comment by yoyojacky** [comment]
> Aah, sorry, I misread a bit. When I had that problem in the beginning, it turned out that the four power pins from the main board had missed their correct holes in the daughter board. But you are probably not as clumsy as me...
> 
> It's definitely possible that your board is dead. I thought that several times during the most stressful 48 hours of my recent life.... And it still ended up being a software/config problem in my case.

Dose your deskpi work properly after correcting installation?

**Comment by yoyojacky** [comment]
> > > Took off the rear panel, managed to get it squeezed on, did that a few times, replaced the panel and it now seems a little happier.
> > > Maybe revisit this bit of the design as the tolerances are really, really tight.
> > 
> > 
> > Are you saying that the gap between the acrylic shell and the aluminum alloy shell is too small?
> 
> No, the row of pins nearest the PCB have an incredibly tight tolerance making it hard to plug in a standard ribbon cable.
> 
> It's nothing to do with the alu case or acrylic panels.



> > Edit: here is a current screenshot. Status is "COMPLETED", not "REFUNDED"
> > ![image](https://user-images.githubusercontent.com/68641750/117949606-970e1d80-b312-11eb-9c6e-5f14cf0d6a53.png)
> 
> @yoyojacky
> It is really ridiculous there is still no answer. Either refund me immediately, or send me the full package of a case with a working board (UASP / TRIM support included!), a32GB SD-card and bracket, as promised.
> I only cancelled since the first board was faulty and the second had no UASP / TRIM support. Since that has now changed, I am still interested in the product.

Please check your paypal account, we have already refund for a long time....
![image](https://user-images.githubusercontent.com/6893075/123935040-72572d80-d9c6-11eb-92f9-dc766a4bccd5.png)

**Comment by Mister-42** [comment]
Yes you did. But that was 6 days AFTER the message you quote. So at the time of my writing, that was absolutely correct.
